src/app/meshes/data/index.spec.ts

```diff
@@ -99,6 +99,9 @@ describe('meshes data transformations', () => {
               },
             ],
           },
+          routing: {
+            zoneEgress: false,
+          },
```

Comment:
I added a check in the above `fromObject` code to set `routing.zoneEgress` to `false` if routing is completely `undefined`. In the end we didn't need this here, but we use this pattern a lot and we might need this at some point later so I left it in.